### DISCUSSION TITLE
Pianoroll notes to instruments modules ?

### DISCUSSION THREAD

**Discussion by subderground** [DISTRHO/Cardinal#720] [Q&A]
Hello, The Pianoroll that comes with the 24 09 release is a gamechanger. 🤩 :-)
Now I try to send notes to différents modules (sampler/osc). ie : F2 to osc01, D2 to osc02, A2 to sampler01, C2 to kick, C3->C4 to sampler02...  Have Cardinal a module(s) to do that ?  Many thanks.

**Comment by subderground** [comment]
I think Orangeline Bukets module can do that but it's not yet into Cardinal.😇

**Reply by dromer** [reply]
That module is `GPL-3.0-only` so won't ever come to Cardinal :#

**Comment by rl2939** [comment]
This is a bit late, but would [Notedetector by Voxglitch](https://github.com/clone45/voxglitch/blob/master/docs/note-detector.md) be what you are looking for?
